pkgs/shells/zsh/zsh-you-should-use/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-you-should-use";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "MichaelAquilina";
+    repo = pname;
+    rev = version;
+    sha256 = "0fig5ralagi5jajk7gdm52jvwql17qk9cd6j98qsndvckb26a753";
+  };
+
+  dontBuild = true;
```

Comment:
Does using ``zunit`` makes any sense for us?

Reply:
(on a side-note: `dontBuild` isn't used to disable tests).

But do you think it is? It doesn't seem too widely adopted :)

Reply:
Sorry, my brainfart :) I read ``dontCheck`` and was checking the zunit tests in there.